INPTPZ1/Program.cs

```diff
@@ -23,252 +23,321 @@ namespace INPTPZ1
     /// </summary>
     class Program
     {
+
         static void Main(string[] args)
         {
-            int[] intargs = new int[2];
-            for (int i = 0; i < intargs.Length; i++)
+
+            ArgsAdapter.initArgs(args);
+            FractalViewer fractalViewer = new FractalViewer();
+            fractalViewer.DrawImage();
+            Console.ReadKey();
+        }
+    }
+
+    namespace Mathematics
+    {
+
+        class ArgsAdapter
+        {
+            private static int[] intargs;
```

Comment:
Nevhodné řešení - na první pohled se to tváří jako obecný konvertor, ale je to napsané na míru konkrétnímu použití a přitom to používá nic neříkající pojmenování jako je "intargs" a "doubleargs". Doporučuji předělat lépe.